--- setuptools-65.3.0/setuptools/_distutils/sysconfig.py.orig	2022-08-31 15:40:59.444208654 +0200
+++ setuptools-65.3.0/setuptools/_distutils/sysconfig.py	2022-08-31 15:42:02.547200129 +0200
@@ -237,13 +237,13 @@ def get_python_lib(plat_specific=0, stan
             prefix = plat_specific and EXEC_PREFIX or PREFIX
 
     if os.name == "posix" or is_mingw():
-        if plat_specific or standard_lib:
+        if plat_specific or standard_lib or prefix != "/usr":
             # Platform-specific modules (any module from a non-pure-Python
             # module distribution) or standard Python library modules.
             libdir = getattr(sys, "platlibdir", "lib")
         else:
             # Pure Python
-            libdir = "lib"
+            libdir = "share"
         implementation = 'pypy' if IS_PYPY else 'python'
         libpython = os.path.join(prefix, libdir, implementation + get_python_version())
         return _posix_lib(standard_lib, libpython, early_prefix, prefix)
