--- setuptools-62.0.0/docs/conf.py.orig	2022-04-04 12:27:45.000000000 +0200
+++ setuptools-62.0.0/docs/conf.py	2022-04-09 10:50:38.582612780 +0200
@@ -84,17 +84,6 @@
 # Be strict about any broken references:
 nitpicky = True
 
-# Include Python intersphinx mapping to prevent failures
-# jaraco/skeleton#51
-extensions += ['sphinx.ext.intersphinx']
-intersphinx_mapping = {
-    'python': ('https://docs.python.org/3', None),
-}
-
-intersphinx_mapping.update({
-    'pypa-build': ('https://pypa-build.readthedocs.io/en/latest/', None)
-})
-
 # Add support for linking usernames
 github_url = 'https://github.com'
 github_repo_org = 'pypa'
@@ -170,12 +159,6 @@
     ('py:mod', 'docutils'),  # there's no Sphinx site documenting this
 ]
 
-# Allow linking objects on other Sphinx sites seamlessly:
-intersphinx_mapping.update(
-    python2=('https://docs.python.org/2', None),
-    python=('https://docs.python.org/3', None),
-)
-
 # Add support for the unreleased "next-version" change notes
 extensions += ['sphinxcontrib.towncrier']
 # Extension needs a path from here to the towncrier config.
@@ -206,10 +189,3 @@
     },
     # rel="apple-touch-icon" does not support SVG yet
 ]
-
-intersphinx_mapping['pip'] = 'https://pip.pypa.io/en/latest', None
-intersphinx_mapping['PyPUG'] = ('https://packaging.python.org/en/latest/', None)
-intersphinx_mapping['packaging'] = ('https://packaging.pypa.io/en/latest/', None)
-intersphinx_mapping['importlib-resources'] = (
-    'https://importlib-resources.readthedocs.io/en/latest', None
-)
